### DISCUSSION TITLE
super responsive table

### DISCUSSION THREAD

**Discussion by ghost** [huntabyte/shadcn-svelte#862] [Help]
is there a way to get the data table to behave like this?: https://github.com/coston/react-super-responsive-table

basically the rows turn into divs on mobile devices, so people dont have to scroll horizontally

**Comment by xob0t** [comment]
shadcn-svelte uses [Svelte Headless Table](https://github.com/bryanmylee/svelte-headless-table), its better to ask questions about it there
